sklearn/linear_model/_passive_aggressive.py

```diff
@@ -231,6 +231,10 @@ def partial_fit(self, X, y, classes=None):
             and can be omitted in the subsequent calls.
             Note that y doesn't need to contain all labels in `classes`.
 
+        sample_weight : None
+            Ignored. This parameter exists only for compatibility with
+            sklearn.linear_model._stochastic_gradient.BaseSGDClassifier.
```

Comment:
I think not having `sample_weight` here is on purpose. The child class does not explicit support sample weights so the child class overrides the parent method to remove the `sample_weight` kwarg.

For scikit-learn, an estimator only defines `sample_weight` if it actually makes use of it.

Also `BaseSGDClassifier` is not a public class, so from a user side there is no compatibility issue.